qiskit-qir/src/qiskit_qir/translate.py

```diff
@@ -0,0 +1,20 @@
+##
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+##
+from qiskit_qir.elements import QiskitModule
+from qiskit_qir.visitor import BasicQisVisitor
+
+
+def to_qir(circuit):
+    module = QiskitModule.from_quantum_circuit(circuit=circuit)
+    visitor = BasicQisVisitor()
+    module.accept(visitor)
+    return visitor.ir()
+
+
+def to_qir_bitcode(circuit):
```

Comment:
The suffix being present here and not on the other function makes it sound like bitcode is more specialized or specific, but both the text and bitcode formats are valid serialization formats for LLVM IR; perhaps it may make sense to modify names to emphasize that?